signatures/signatures.go

```diff
@@ -97,3 +97,19 @@ func GetAllCerts() (types.CertList, error) {
 func isValidSignature(sign util.EFIGUID) bool {
 	return sign == signature.CERT_X509_GUID
 }
+
+// GetEfiCertsCommonNames returns a simple list of the Common names of the certs
+func GetEfiCertsCommonNames() types.EfiCerts {
+	var data types.EfiCerts
+	certs, _ := GetAllCerts() // Ignore errors here, we dont care about them
```

Comment:
I dont know about that. This returns the certs if it can get them, otherwise its an empty eficerts. There is no promise here of returning errors or not, debugging and such, this is mostly an shortcut to not have to do the same thing in the state when getting the names and such, so its a commodity that promises nothing else in the signature of the call.

For getting the actual errors, and the full certificates (not just a list of the common names) there is another function that actually returns an error.

Maybe this should not be public, or maybe I should just move all the mangling into the state instead....

Reply:
updated, now its a commodity method in state just to clean up the actual stuff coming from the signatures